### ISSUE TITLE
Transfer this repo to the JuliaParallel GitHub org?

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaParallel/SlurmClusterManager.jl#22]
@kleinhenz Thanks so much for creating this package. I've been using it, and it solves a lot of problems for me.

Would you be open to transferring this repo to the JuliaParallel GitHub organization? This would make it easier for some more people to participate in maintaining this package.

**Comment by kleinhenz** [comment]
Hi @DilumAluthge. Thanks for all your work on this recently! Transferring to JuliaParallel makes sense to me. I’m currently traveling but I will take a closer look at all the PRs/transfer when I am back in a couple days.

**Comment by DilumAluthge** [comment]
BTW I've invited you to the org:

> You've invited Joseph Kleinhenz to JuliaParallel! They'll be receiving an email shortly. They can also visit https://github.com/JuliaParallel to accept the invitation.

**Comment by DilumAluthge** [comment]
@kleinhenz Shall we transfer this repo to the JuliaParallel org?

**Comment by kleinhenz** [comment]
Do you know if this will cause any problems with registrator?

**Comment by DilumAluthge** [comment]
> Do you know if this will cause any problems with registrator?

Nope, we'll just edit the registry (https://github.com/JuliaRegistries/General/blob/master/S/SlurmClusterManager/Package.toml) as soon as we transfer the repo, and then Registrator will keep working.

**Comment by DilumAluthge** [comment]
> > Do you know if this will cause any problems with registrator?
> 
> Nope, we'll just edit the registry (https://github.com/JuliaRegistries/General/blob/master/S/SlurmClusterManager/Package.toml) as soon as we transfer the repo, and then Registrator will keep working.

https://github.com/JuliaRegistries/General/pull/123763

**Comment by DilumAluthge** [comment]
@kleinhenz Bumping this again.

**Comment by DilumAluthge** [comment]
Thank you @kleinhenz!
